### DISCUSSION TITLE
美國科學家發明「烙賽偵測器」：偷聽你上廁所的聲音

### DISCUSSION THREAD

**Discussion by Tuoresa** [1101-GS4719-C/Python-Community-GS4719-C#1069] [General]
在第75屆美國物理學會「流體力學部」年會中，喬治亞理工學院（Georgia Tech）的科學家昭告世人：他們發明了一個可以「每天關心您腸胃健康」的全自動智慧健檢機器。收集使用中的廁所裡的聲音，大概可以分成「尿尿」、「正常便便」、「烙賽」、「放屁」這四大「風格」，研究者將之比擬為巴洛克時代著名的音樂家安東尼奧．韋瓦第（Antonio Luci Vivaldi, 1678-1741）的曠世巨作「四季」，在前述的美國物理學會年會中，發表了以「[Urination, defecation, flatulence, and diarrhea: the four seasons of toilet acoustics](https://meetings.aps.org/Meeting/DFD22/Session/Z07.6)」（尿尿、便便、脹氣、烙賽：廁所之聲「四季」）的論文

也太神奇！
是說這樣著實是有一點害羞欸 救命哈哈
不過確實有看過許多相關文獻表明
透過觀察及檢測排泄、排遺好像真的能得知身體很多資訊，
(想起了古老的蟯蟲測驗哈哈哈)
大家都要好好上廁所哇！

https://www.thenewslens.com/article/178527

**Comment by wenchi9023** [comment]
我的天 在馬桶上裝五支攝影機監測你上廁所??!!
原本不便秘的人都變成不再上廁所了吧
也太害羞...上個廁所也好緊張...

**Comment by yuxinlin93** [comment]
裝監測器廳上廁所的聲音也太害羞了吧，感覺每次上廁所都有一個人拿者麥克風對準收音，然後研究團隊在旁邊研究聲音，太可怕了吧。但感覺的確還是會有些實質幫助，發現一些平常真的不會注意到的事情，有疾病也比較容易被發現

**Comment by harrylee1971** [comment]
有點小噁心(?因為醫學檢驗靠這樣子也沒辦法達成精準的判斷，只能從小方面去略知一二而已

**Comment by BlahFish1006** [comment]
好好笑，感覺是個很有味道的研究 XD 
我只能說研究者們的實驗精神真的十分令人欽佩啊。
而且文章中還有提到，「烙賽」的辨識正確率竟高達98%！即使同時有人講話的雜訊，正確率也有96%，真的太厲害了......
雖然這研究看似蠻搞笑且無理頭，但事實上現代人的腸胃問題也值得被更加重視，
台灣人罹患大腸癌的比例目前更是居世界之冠！這個研究若能幫助潛在患者提早有警覺，我覺得也非常有意義。

**Comment by Sylvie0601** [comment]
好好笑，我覺得比擬為韋瓦第的"四季"超級好笑的哈哈哈 ! 但其實腸胃健康是蠻多人應該要注重的ㄟ，感覺身邊很多人都有腸胃問題，如果在未來有特別應用在醫療上面感覺其實不賴ㄟ !

**Comment by Guagua1234** [comment]
這個發明也太害羞了吧，但真的有時候需要透過分辨自己的腸胃狀況來了解自己的健康，而且要如何從放屁狀況知道自己的資訊，真令人感到好奇
